portfolio/src/main/webapp/script.js

```diff
@@ -44,3 +44,8 @@ function addFact() {
 
 }
 
+async function getHelloWorldText() {
```

Comment:
Do we want this function to be asynchronous?
If so, sounds good.
If not, we can make this function synchronous by removing the async modifier and the await modifiers below.